components/inputs/input-date.js

```diff
@@ -14,7 +14,6 @@ import { getUniqueId } from '../../helpers/uniqueId.js';
 import { ifDefined } from 'lit/directives/if-defined.js';
 import { LabelledMixin } from '../../mixins/labelled-mixin.js';
 import { LocalizeCoreElement } from '../../helpers/localize-core-element.js';
-import ResizeObserver from 'resize-observer-polyfill/dist/ResizeObserver.es.js';
```

Comment:
Do you remember why this was using the polyfill? Are we at the point where we can kill the polyfill everywhere?

Reply:
What happens if you try to use it Safari pre-13.1 without feature detection?  Does it blow up?

Reply:
Blows up with:
> ReferenceError: can't find variable: ResizeObserver